apps/wakunode2/wakunode2.nim

```diff
@@ -709,12 +710,13 @@ when isMainModule:
   # Handle SIGSEGV
   when defined(posix):
     proc handleSigsegv(signal: cint) {.noconv.} =
-      fatal "Shutting down after receiving SIGSEGV"
-      waitFor node.stop()
+      # Require --debugger:native
+      fatal "Shutting down after receiving SIGSEGV", stacktrace=getBacktrace()
 
-      # Only available with --stacktrace:on --linetrace:on
+      # Not available in -d:release mode
       writeStackTrace()
```

Comment:
For interest's sake. If not in `-d:release` I imagine we'll now print the libbacktrace and stacktrace. Are both necessary/helpful?